### DISCUSSION TITLE
Wrong rendering for Tables' shadow in light theme

### DISCUSSION THREAD

**Discussion by santiagxf** [pradyunsg/furo#180] [Potential Issue] [answered]
When tables render in light theme, the table's shadow take 100% of available width even when the table doesn't not. This issue can be seen even in the Furo's documentation page.

![image](https://user-images.githubusercontent.com/32112894/124642034-bb473000-de65-11eb-868f-f9cf2f446ebf.png)

**Comment by pradyunsg** [comment]
Buggery. Nice catch. Moar Sphinx 4 markup changes. 💁🏽

**Reply by pradyunsg** [reply]
Nope -- I broke this in https://github.com/pradyunsg/furo/commit/364daf932f09c7ea7264ff5004f468328b9e6058. :)

**Comment by pradyunsg** [comment, marked as answer]
Filed #183 to track fixing this issue!
